pangeo_forge_runner/commands/bake.py

```diff
@@ -168,6 +169,21 @@ def autogenerate_job_name(self):
 
         return job_name
 
+    def add_unique_suffix_to_flink_jobs(self, per_recipe_unique_job_name):
```

Comment:
In another PR, we should probably go through and figure out which behaviors we can expect runner implementations to sort out so that `Bake` can be thin and runner independent. It'd be great for it to not know anything at all about implementation details